src/main/java/chess/web/controller/ChessGameRestController.java

```diff
@@ -0,0 +1,57 @@
+package chess.web.controller;
+
+import chess.web.dto.CreateRoomRequestDto;
+import chess.web.dto.CreateRoomResultDto;
+import chess.web.dto.DeleteDto;
+import chess.web.dto.DeleteResultDto;
+import chess.web.dto.MoveDto;
+import chess.web.dto.MoveResultDto;
+import chess.web.dto.PlayResultDto;
+import chess.web.dto.ReadRoomResultDto;
+import chess.web.dto.RoomDto;
+import chess.web.service.ChessGameService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ChessGameRestController {
+
+    private final ChessGameService service;
+
+    public ChessGameRestController(ChessGameService service) {
+        this.service = service;
+    }
+
+    @GetMapping("/start/{roomId}")
+    public RoomDto start(@PathVariable("roomId") int roomId) {
```

Comment:
```suggestion
    public RoomDto start(@PathVariable int roomId) {
```
참고로 `@PathVariable`은 쿼리 파라미터의 이름과 파라미터 이름이 동일하다면 `@PathVariable`의 value를 생략할 수 있어요. `@RequestParam` 등도 동일하게 기능을 제공하는데요. 참고차 남깁니다.

Reply:
감사합니다. 공식 문서까지 확인해봤습니다!